templates/android-dot-com/index.html

```diff
@@ -86,7 +86,7 @@
         </div>
       </div>
 
-      <div class="android-drawer mdl-layout__drawer">
+      <div class="android-drawer mdl-layout__drawer" aria-hidden="true">
```

Comment:
Does this attribute need to be set by default for this to work properly on load? I'd prefer it if we set this automatically on initialization for existing integrations to have no changes to benefit from the  improvements.